example/server/model-config.json

```diff
@@ -1,32 +1,6 @@
 {
-  "_meta": {
-    "sources": [
-      "loopback/common/models",
-      "loopback/server/models",
-      "../common/models",
-      "./models"
-    ],
-    "mixins": [
-      "loopback/common/mixins",
-      "loopback/server/mixins",
-      "../common/mixins",
-      "./mixins"
-    ]
-  },
-  "Client": {
-    "dataSource": "postgresql",
-    "public": true
-  },
-  "Roles": {
-    "dataSource": "postgresql",
-    "public": true
-  },
   "Users": {
-    "dataSource": "postgresql",
-    "public": true
-  },
-  "RoleUsers": {
-    "dataSource": "postgresql",
+    "dataSource": null,
```

Comment:
this is the first iteration and the model only reads a file and does not try to cache in a db.
so there is no need of a db now. later we would add a "datagraph" as a db in the next iteration.